js/libs/keycloak-admin-client/README.md

```diff
@@ -123,6 +123,27 @@ Then start the tests with:
 pnpm test
 ```
 
+## methods
+
+### registerTokenProvider
+
+The term "keycloakAdminClient" refers to an instance of the Keycloak Admin Client library. Keycloak is an open-source identity and access management solution, and the Admin Client is a component that provides programmatic access to the administrative functionalities of a Keycloak server.
+
+In the code:
+
+```
+keycloakAdminClient.registerTokenProvider({
+  getAccessToken: () => Promise.resolve('some access token'),
+});
+```
```

Comment:
Let's add some markup so this interpreted as JavaScript, also let's be more explicit about returns for the sake of making things easier to understand for novice users.
```suggestion
```js
keycloakAdminClient.registerTokenProvider({
  getAccessToken() {
    return Promise.resolve('some access token');
  }
});
```
```